From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 17 Dec 2012 22:54:15 +0100
Subject: [PATCH] Release raspberrypi/20121217

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 686b83c..0240f39 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 6
 SUBLEVEL = 10
-EXTRAVERSION =
+EXTRAVERSION =-20121217
 NAME = Terrified Chipmunk
 
 # *DOCUMENTATION*
