Index: linux-3.18/arch/arm/boot/dts/kirkwood-dockstar.dts
===================================================================
--- linux-3.18.orig/arch/arm/boot/dts/kirkwood-dockstar.dts
+++ linux-3.18/arch/arm/boot/dts/kirkwood-dockstar.dts
@@ -32,6 +32,17 @@
 				marvell,function = "gpio";
 			};
 		};
+		i2c@11000 {
+			status = "okay";
+			clock-frequency = <400000>;
+			pinctrl-0 = <&pmx_twsi0>;
+			pinctrl-names = "default";
+
+			bmp085: bmp085@77 {
+				compatible = "bmp085";
+				reg = <0x77>;
+			};
+		};
 		serial@12000 {
 			status = "ok";
 		};
