From: popcornmix <popcornmix@gmail.com>
Date: Tue, 11 Dec 2012 19:04:27 +0000
Subject: [PATCH] Increase default coherent pool so vchiq starts up

---
 arch/arm/mach-bcm2708/bcm2708.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/mach-bcm2708/bcm2708.c b/arch/arm/mach-bcm2708/bcm2708.c
index 9f456e9..87fd348 100644
--- a/arch/arm/mach-bcm2708/bcm2708.c
+++ b/arch/arm/mach-bcm2708/bcm2708.c
@@ -850,9 +850,7 @@ void __init bcm2708_init_early(void)
 	 * context. Increase size of atomic coherent pool to make sure such
 	 * the allocations won't fail.
 	 */
-#if !defined(CONFIG_BCM_VC_CMA)
-	init_dma_coherent_pool_size(SZ_2M);
-#endif
+	init_dma_coherent_pool_size(SZ_4M);
 }
 
 static void __init board_reserve(void)
