---
 arch/arm/mach-kirkwood/sheevaplug-setup.c |   12 ++++++++++++
 1 file changed, 12 insertions(+)

--- a/arch/arm/mach-kirkwood/sheevaplug-setup.c
+++ b/arch/arm/mach-kirkwood/sheevaplug-setup.c
@@ -15,6 +15,7 @@
 #include <linux/mtd/partitions.h>
 #include <linux/mv643xx_eth.h>
 #include <linux/gpio.h>
+#include <linux/i2c.h>
 #include <linux/leds.h>
 #include <asm/mach-types.h>
 #include <asm/mach/arch.h>
@@ -88,6 +89,8 @@ static unsigned int sheevaplug_mpp_confi
 	MPP29_GPIO,	/* USB Power Enable */
 	MPP46_GPIO,	/* LED Red */
 	MPP49_GPIO,	/* LED */
+	MPP8_TW0_SDA,	/* I2C data */
+	MPP9_TW0_SCK,	/* I2C clock */
 	0
 };
 
@@ -99,6 +102,12 @@ static unsigned int sheeva_esata_mpp_con
 	0
 };
 
+static struct i2c_board_info __initdata i2c_board_info[] = {
+	{
+		I2C_BOARD_INFO("bmp085", 0x77),
+	},
+};
+
 static void __init sheevaplug_init(void)
 {
 	/*
@@ -133,6 +142,9 @@ static void __init sheevaplug_init(void)
 		kirkwood_sdio_init(&sheevaplug_mvsdio_data);
 
 	platform_device_register(&sheevaplug_leds);
+
+	i2c_register_board_info(0, i2c_board_info, ARRAY_SIZE(i2c_board_info));
+	kirkwood_i2c_init();
 }
 
 #ifdef CONFIG_MACH_SHEEVAPLUG
