From: Bernhard Walle <bernhard@bwalle.de>
Date: Sun, 19 Feb 2012 15:20:48 +0100
Subject: [PATCH] Dockstar: Don't allow to save the envionrment in NAND

Signed-off-by: Bernhard Walle <bernhard@bwalle.de>
---
 include/configs/dockstar.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/include/configs/dockstar.h b/include/configs/dockstar.h
index cf82ea0e1373..ad87f8cb4b44 100644
--- a/include/configs/dockstar.h
+++ b/include/configs/dockstar.h
@@ -65,12 +65,7 @@
 /*
  *  Environment variables configurations
  */
-#ifdef CONFIG_CMD_NAND
-#define CONFIG_ENV_IS_IN_NAND		1
-#define CONFIG_ENV_SECT_SIZE		0x20000	/* 128K */
-#else
 #define CONFIG_ENV_IS_NOWHERE		1	/* if env in SDRAM */
-#endif
 /*
  * max 4k env size is enough, but in case of nand
  * it has to be rounded to sector size
